<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MembersTab</class>
 <widget class="QWidget" name="MembersTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTop">
     <item><widget class="QPushButton" name="buttonAddMember"><property name="text"><string>Ajouter</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonEditMember"><property name="text"><string>Modifier</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonDeleteMember"><property name="text"><string>Supprimer</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonExportMembers"><property name="text"><string>Exporter</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonMailing"><property name="text"><string>Ouvrir Mailing</string></property></widget></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSummary">
     <item>
      <widget class="QLabel" name="labelTotalAmount">
       <property name="text"><string>Total général : 0.00 €</string></property>
       <property name="styleSheet"><string>QLabel { font-weight: bold; color: #2E7D32; padding: 5px; }</string></property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCashAmount">
       <property name="text"><string>Espèces : 0.00 €</string></property>
       <property name="styleSheet"><string>QLabel { padding: 5px; color: #1976D2; }</string></property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChecksAmount">
       <property name="text"><string>Chèques : 0.00 €</string></property>
       <property name="styleSheet"><string>QLabel { padding: 5px; color: #1976D2; }</string></property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAncvAmount">
       <property name="text"><string>ANCV : 0.00 €</string></property>
       <property name="styleSheet"><string>QLabel { padding: 5px; color: #1976D2; }</string></property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableMembers">
     <column>
      <property name="text"><string>Nom</string></property>
     </column>
     <column>
      <property name="text"><string>Prénom</string></property>
     </column>
     <column>
      <property name="text"><string>Adresse</string></property>
     </column>
     <column>
      <property name="text"><string>CP</string></property>
     </column>
     <column>
      <property name="text"><string>Ville</string></property>
     </column>
     <column>
      <property name="text"><string>Tél</string></property>
     </column>
     <column>
      <property name="text"><string>Mail</string></property>
     </column>
     <column>
      <property name="text"><string>Date naissance</string></property>
     </column>
     <column>
      <property name="text"><string>RGPD</string></property>
     </column>
     <column>
      <property name="text"><string>Droit image</string></property>
     </column>
     <column>
      <property name="text"><string>Espèce</string></property>
     </column>
     <column>
      <property name="text"><string>Chèque 1</string></property>
     </column>
     <column>
      <property name="text"><string>Chèque 2</string></property>
     </column>
     <column>
      <property name="text"><string>Chèque 3</string></property>
     </column>
     <column>
      <property name="text"><string>ANCV</string></property>
     </column>
     <column>
      <property name="text"><string>Total</string></property>
     </column>
     <column>
      <property name="text"><string>Club MJC</string></property>
     </column>
     <column>
      <property name="text"><string>Autres clubs MJC</string></property>
     </column>
     <column>
      <property name="text"><string>Statut cotisation</string></property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFilter">
     <item><widget class="QPushButton" name="buttonFilter"><property name="text"><string>Filtrer</string></property></widget></item>
     <item><widget class="QPushButton" name="buttonResetFilter"><property name="text"><string>Réinitialiser</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>